.gitignore

```diff
@@ -7,7 +7,7 @@ tmp/
 log/
 logs/
 !vendor/travis-core/lib/travis/logs/
-!vendor/travis-core/lib/travis/model/log/
+!/lib/travis/model/log/
```

Comment:
i don't think where this came from, but pretty sure we don't want to ignore anything in `lib`? the previous one was `logs`, now it's `log`. i think we can just remove this line?